Formula/dotdrop.rb

```diff
@@ -3,8 +3,8 @@ class Dotdrop < Formula
 
   desc "Save your dotfiles once, deploy them everywhere"
   homepage "https://deadc0de.re/dotdrop"
-  url "https://github.com/deadc0de6/dotdrop/archive/refs/tags/v1.8.2.tar.gz"
-  sha256 "4e81db35e47c05607e46472e2919f0840e4ec0a1c168214f0bff54b876245340"
+  url "https://github.com/deadc0de6/dotdrop/archive/refs/tags/v1.9.0.tar.gz"
+  sha256 "5e79deb64322fcd4f745734e8fcfa293e2f659738599012844ccd57b13592f57"
```

Comment:
Noting that if this is switched to PyPI URL, then the resources can be auto-updated as part of `brew bump` or `brew bump-formula-pr` (possibly via one of the GitHub actions)
 
https://files.pythonhosted.org/packages/16/56/5fa22cdd89f55773250a675c940e18b2188ebe2511762347d0bf0b3c9841/dotdrop-1.9.0.tar.gz

Reply:
good to know, will do on next update, thanks!